Diagnostics/FreeBusyChecker/Functions/OnPremOAuthFunctions.ps1

```diff
@@ -0,0 +1,654 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', '', Justification = 'Variables are being used in functions')]
+param (
+    [Parameter(Mandatory = $false)]
+    [String] $tdIntraOrgTarGetAddressDomain
+)

```

Comment:
Not needed

Reply:
Removed from code. Correction will be available when changes are pushed.